### PR TITLE
Add guide for Azure Postgres/MySQL database access

### PR DESCRIPTION
**Pull request by r0mant**
This PR adds a new database access guide page that explains how to setup database access with Azure-hosted Postgres or MySQL databases. The actual functionality was merged in master (https://github.com/gravitational/teleport/pull/8951) and v8 (https://github.com/gravitational/teleport/pull/9185) a few weeks ago so it's already supported in the product, just the guide has been missing.

### CODE REVIEW

**Comment by r0mant** [comment]
@klizhentas @ptgott @xinding33 Could you guys review this please? I'd like to deploy this sooner than later so people can start using database access with Azure Postgres/MySQL. Thanks!

File: docs/pages/database-access/guides/azure-postgres-mysql.mdx [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+  *Add a certificate or secret*:
+
+  ![Registered app](../../../img/database-access/guides/azure/registered-app@2x.png)
+
+  Create a new client secret that Database Service agent will use to
+  authenticate with Azure API:
+
+  ![Registered app secrets](../../../img/database-access/guides/azure/registered-app-secrets@2x.png)
+
+  Teleport Database Service uses Azure SDK default credential provider chain to
```
Comment (ptgott) [inline docs/pages/database-access/guides/azure-postgres-mysql.mdx:151]:
```suggestion
  Teleport Database Service uses Azure SDK's default credential provider chain to
```

File: docs/pages/database-access/guides/azure-postgres-mysql.mdx [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+  look for credentials. Refer to [Authentication methods](https://docs.microsoft.com/en-us/azure/developer/go/azure-sdk-authorization)
+  to pick a method suitable for your use-case. For example, to use
+  environment-based authentication with client secret, Database Service should
```
Comment (ptgott) [inline docs/pages/database-access/guides/azure-postgres-mysql.mdx:154]:
```suggestion
  environment-based authentication with a client secret, Database Service should
```

File: docs/pages/database-access/guides/azure-postgres-mysql.mdx [hunk 3]
```diff
@@ -0,0 +1,276 @@
...
+</Tabs>
+
+## Step 3/4. Create Azure database users
+
+To let Teleport connect to your Azure database authenticating as a service
+principal you need to create Azure AD users for that principal in the database.
+
+### Assign Azure AD administrator
+
+Only Azure AD administrator for the database can connect to it and create
```
Comment (ptgott) [inline docs/pages/database-access/guides/azure-postgres-mysql.mdx:172]:
```suggestion
Only the Azure AD administrator for the database can connect to it and create
```

File: docs/pages/database-access/guides/azure-postgres-mysql.mdx [hunk 4]
```diff
@@ -0,0 +1,276 @@
...
+subsequent Azure AD users. Go to your database _Active Directory admin_ page
```
Comment (ptgott) [inline docs/pages/database-access/guides/azure-postgres-mysql.mdx:173]:
```suggestion
Azure AD users. Go to your database's _Active Directory admin_ page
```

File: docs/pages/database-access/guides/azure-postgres-mysql.mdx [hunk 5]
```diff
@@ -0,0 +1,276 @@
...
+and set the AD admin using _Set admin_ button:
```
Comment (ptgott) [inline docs/pages/database-access/guides/azure-postgres-mysql.mdx:174]:
```suggestion
and set the AD admin using the _Set admin_ button:
```

File: docs/pages/database-access/guides/azure-postgres-mysql.mdx [hunk 6]
```diff
@@ -0,0 +1,276 @@
...
+
+![Set AD admin](../../../img/database-access/guides/azure/set-ad-admin@2x.png)
+
+Note that only one AD user can be set as an admin for the database.
+
+### Connect to database as AD admin
```
Comment (ptgott) [inline docs/pages/database-access/guides/azure-postgres-mysql.mdx:180]:
```suggestion
### Connect to the database as an AD admin
```

File: docs/pages/database-access/guides/azure-postgres-mysql.mdx [hunk 7]
```diff
@@ -0,0 +1,276 @@
...
+
+Next, you need to connect to your database as the AD admin user.
+
+Use Azure `az` CLI utility to log in as the user that you set as the AD admin,
```
Comment (ptgott) [inline docs/pages/database-access/guides/azure-postgres-mysql.mdx:184]:
```suggestion
Use the Azure `az` CLI utility to log in as the user that you set as the AD admin,
```

File: docs/pages/database-access/guides/azure-postgres-mysql.mdx [hunk 8]
```diff
@@ -0,0 +1,276 @@
...
+</TabItem>
+</Tabs>
+
+Note that the database username must include `@instance-name` suffix with the
+name of the Azure database instance you're connecting to.
+
+### Create AD users
+
+Once connected to the database as AD admin, create database users for the
+service principal Teleport Database Service will be using. Use `Client ID`
```
Comment (ptgott) [inline docs/pages/database-access/guides/azure-postgres-mysql.mdx:210]:
```suggestion
service principal that Teleport Database Service will be using. Use `Client ID`
```

File: docs/pages/database-access/guides/azure-postgres-mysql.mdx [hunk 9]
```diff
@@ -0,0 +1,276 @@
...
+  Query OK, 0 rows affected (0.92 sec)
+  ```
+</TabItem>
+</Tabs>
+
+You can create multiple database users for the same service principal.
+
+## Step 4/4. Connect
+
+Log into your Teleport cluster, your Azure database should appear in the list of
```
Comment (ptgott) [inline docs/pages/database-access/guides/azure-postgres-mysql.mdx:236]:
```suggestion
Log into your Teleport cluster. Your Azure database should appear in the list of
```

File: docs/pages/database-access/guides/azure-postgres-mysql.mdx [hunk 10]
```diff
@@ -0,0 +1,276 @@
...
+
+Now connect to the database:
+
+```code
+$ tsh db connect azure-db
+```
+
+<Admonition type="note">
+  The appropriate database command-line client (`psql`, `mysql`) should be
+  available in PATH in order to be able to connect.
```
Comment (ptgott) [inline docs/pages/database-access/guides/azure-postgres-mysql.mdx:265]: Does this refer to the end-user's machine or the VM running the Database Service?

**Comment by r0mant** [comment]
@ptgott Thank you for the review! I have addressed your feedback, could you take another look?

**Review by ptgott — APPROVED** [review]

**Comment by r0mant** [comment]
@klizhentas @xinding33 Need a second review on this from either of you. Could you take a look please?

**Review by xinding33 — APPROVED** [review]

**Pull request merged**